controller-server/src/test/java/com/yahoo/vespa/hosted/controller/deployment/ApplicationPackageBuilder.java

```diff
@@ -99,6 +99,13 @@ public ApplicationPackageBuilder region(String regionName) {
         return this;
     }
 
+    public ApplicationPackageBuilder test(String regionName) {
+        environmentBody.append("      <test>");
```

Comment:
Have we decided on this tag name? `<test>` already means something else.

Reply:
Yes, it was through a review.